### PR TITLE
option to list steps executed

### PR DESCRIPTION
**Pull request by manishtomar**
if convergence is triggered on the group when desired is set to actual. This should be run before enabling convergence for tenants to ensure no changes is made in their infra.

### CODE REVIEW

File: scripts/trigger_convergence.py [hunk 1]
```diff
@@ -123,6 +136,35 @@ def get_groups(parsed, store, conf):
...
+    Return Effect of list of steps that would be performed on the group
+    if convergence is triggered on it with desired=actual
+    """
+    now_dt = yield Effect(Func(datetime.utcnow))
+    all_data_eff = convergence_exec_data(
+        group["tenantId"], group["groupId"], now_dt, get_executor)
+    all_data = yield Effect(TenantScope(all_data_eff, group["tenantId"]))
+    (executor, scaling_group, group_state, desired_group_state,
+     resources) = all_data
+    desired_group_state.desired = len(resources['servers'])
```
Comment (dragorosson) [inline scripts/trigger_convergence.py]: What if this is a launch_stack group?

Comment (manishtomar) [inline scripts/trigger_convergence.py]: It might raise `KeyError` from `resources["servers"]`. However, I am not concerned with that since this script is only used to transition _existing_ tenants with `launch_server` groups to convergence.

File: scripts/trigger_convergence.py [hunk 2]
```diff
@@ -7,12 +7,18 @@
 Examples:
 `python trigger_convergence -c config.json -g "tenantid:groupId"`
 will trigger convergence on given group(s)
-`python trigger_convergence -c config.json`
+`python trigger_convergence -c config.json --all`
```
Comment (dragorosson) [inline scripts/trigger_convergence.py]: Is this an unrelated change? --all vs --steps

File: scripts/trigger_convergence.py (hunk 1 above)
Comment (dragorosson) [inline scripts/trigger_convergence.py]: Gotcha.

**Comment by dragorosson** [comment]
LGTM :)

File: scripts/trigger_convergence.py (hunk 1 above)
Comment (dragorosson) [inline scripts/trigger_convergence.py]: (That's what I figured this was for.)

File: scripts/trigger_convergence.py (hunk 2 above)
Comment (manishtomar) [inline scripts/trigger_convergence.py]: Yes. Sorry this is correct example after implementing `--all`. Didn't change it then :)

**Comment by manishtomar** [comment]
@dragorosson Thank you soo much for the review!
